nltk/parse/corenlp.py

```diff
@@ -105,7 +105,12 @@ def __init__(
         self.corenlp_options = corenlp_options
         self.java_options = java_options or ['-mx2g']
 
-    def start(self):
+    def start(self, stdout='devnull', stderr='devnull'):
```

Comment:
Will this change start suppressing previously visible output?

Reply:
So you would prefer having 'pipe' as the default?  the problem with it is that the pipe will get filled quickly (for me it was 882 requests with random test until it was full)

Reply:
Ohhh I just now got what you were asking. No, the output was not visible through python before and it has not changed.